addons/mail/static/src/discuss/core/public_web/thread_patch.js

```diff
@@ -0,0 +1,17 @@
+import { Thread } from "@mail/core/common/thread";
+import { patch } from "@web/core/utils/patch";
+
+patch(Thread.prototype, {
+    get orderedMessages() {
+        const result = super.orderedMessages;
+        const { initial_message_id } = this.props.thread;
+        if (initial_message_id && !initial_message_id.isEmpty) {
```

Comment:
Again don't destructure, this is highly discouraged with reactive or any proxy object because you can break its usage: https://vuejs.org/guide/essentials/reactivity-fundamentals.html#limitations-of-reactive

(see 3. point)